File: src/test/java/com/ada/AdaApplicationTests.java
package com.ada;

import static org.junit.jupiter.api.Assertions.assertEquals;

import com.ada.conversation.application.dto.ChatRequest;
import org.junit.jupiter.api.Test;

class AdaApplicationTests {
  @Test
  void createsApplicationRequest() {
    var request = new ChatRequest("hello", null);

    assertEquals("hello", request.message());
  }
}

File: src/test/kotlin/com/ada/AdaApplicationTests.kt
package com.ada

import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest(properties = ["ada.data-dir=target/test-data/application-context"])
class AdaApplicationTests {
    @Test
    fun contextLoads() {
    }
}
